inc/customizer/options/layout_single_post.php

```diff
@@ -158,6 +158,27 @@ private function header_layout() {
 			)
 		);
 
+		$live_refresh_settings = [
+			'responsive' => true,
+			'prop'       => 'min-height',
+			'suffix'     => [
+				'mobile'  => 'px',
+				'desktop' => 'px',
+				'tablet'  => 'px',
+			],
+			'template'   => '.nv-post-cover{ min-height: {{value}}; }',
+		];
+
+		if ( neve_is_new_skin() ) {
```

Comment:
Can't we merge these arrays together? I don't think these if-s are necessary. Same for the others below 😄